src/main/java/org/broadinstitute/hellbender/engine/Shard.java

```diff
@@ -0,0 +1,87 @@
+package org.broadinstitute.hellbender.engine;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * An interface representing a group of reads with an associated
```

Comment:
Docs are corrupted here -- how about `A Shard of records of type T covering a specific genomic interval, optionally expanded by a configurable amount of padded data, that provides the ability to iterate over its records.`